Doc/library/socket.rst

```diff
@@ -661,7 +661,7 @@ The following functions all create :ref:`socket objects <socket-objects>`.
 .. function:: create_server(address, *, family=AF_INET, backlog=None, reuse_port=False, dualstack_ipv6=False)
 
    Convenience function which creates a TCP socket bound to *address* (a 2-tuple
-   ``(host, port)``) and return the socket object.
+   ``(host, port)``) and returns the socket object.
```

Comment:
Note: Other methods in this file are documented using the imperative mood. Rewording this whole section (this method) is out of the scope of this PR, though.